app/src/main/res/drawable/thumb_terraform_cloud.xml

```diff
@@ -0,0 +1,20 @@
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:name="vector"
```

Comment:
Could you remove the `android:name` tags (here and in the 3 paths below). We don't use them in any thumbnails and that can save a few bytes.

Reply:
done!